### PR TITLE
Add missing onLayout prop typings

### PR DESCRIPTION
**Pull request by sinxwal**
# Summary
onLayout prop typings are missing. This PR simply adds missing onLayout prop to svg nodes.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |

## Checklist

- [X] I have tested this on a device and a simulator
- [ ] I added documentation in `README.md`
- [X] I updated the typed files (typescript)
- [ ] I added a test for the API in the `__tests__` folder

### CODE REVIEW

**Review by guiccbr — APPROVED** [review]

File: src/index.d.ts [hunk 1]
```diff
@@ -217,6 +217,10 @@ export interface CommonMarkerProps {
   markerEnd?: string;
 }
 
+export interface LayoutProps {
```
Comment (guiccbr) [inline src/index.d.ts:220]: There are also a few other props missing, as `style`. I wonder if we should use a more generic name instead, as maybe `NativeProps` or `ViewProps` to allow us to add the other missing props here too.

Comment (sinxwal) [reply src/index.d.ts:220]: `NativeProps` sounds good. I'll rename `LayoutProps`

**Comment by sinxwal** [comment]
@guiccbr  ready for merge

**Comment by guiccbr** [comment]
> @guiccbr ready for merge

I can't merge. I'm not a maintainer : (

**Comment by WoLewicki** [comment]
Since #1815, I will close this PR. Thanks for your contribution! :tada:

**Pull request closed without merging**